charts/forkmon/templates/configmap.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "forkmon.fullname" . }}
+  labels:
+    {{- include "forkmon.labels" . | nindent 4 }}
+data:
+  config.toml: |
```

Comment:
I think it would make sense to move the whole config template to values. This would make it fully configurable by the chart user. You can use the `tpl` helper function to render it here directly. Similar to what we have in the dshackle chart in here https://github.com/skylenet/ethereum-helm-charts/blob/master/charts/dshackle/templates/configmap.yaml#L9